examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,58 +14,56 @@
   },
```

Comment:
<p>Again, I don't know if this is reviewnb or the notebook itself, but I see the html repr of the sample_stats dataset before the cell with "NUTS provides the following statistics:". We should keep the same order.</p><p> </p><p>Moreover, here we are only interested in the variable names, not their values, so I'd use <code>trace.sample_stats.data_vars</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>